hack/generate-metallb-manifests.sh

```diff
@@ -11,7 +11,12 @@ METALLB_SC_FILE=$(dirname "$0")/securityContext.yaml
 if ! command -v yq &> /dev/null
 then
     echo "yq binary not found, installing... "
+    yq_tmp_dir=$(mktemp -d)
```

Comment:
I think using `go install` is better here.
See https://tip.golang.org/doc/go1.16#modules
"go install now accepts arguments with version suffixes (for example, go install example.com/cmd@v1.0.0). This causes go install to build and install packages in module-aware mode, ignoring the go.mod file in the current directory or any parent directory, if there is one. This is useful for installing executables without affecting the dependencies of the main module. "

Reply:
nice, will do that